pkg/cidata/cidata.TEMPLATE.d/boot/25-guestagent-base.sh

```diff
@@ -10,6 +10,13 @@ for f in $(seq 0 $((LIMA_CIDATA_MOUNTS - 1))); do
 	mkdir -p "${mountpoint}"
 	gid=$(id -g "${LIMA_CIDATA_USER}")
 	chown "${LIMA_CIDATA_UID}:${gid}" "${mountpoint}"
+
+	mountpointopt="LIMA_CIDATA_MOUNTS_${f}_MOUNTOPTION"
+	option="$(eval echo \$"$mountpointopt")"
```

Comment:
?

Reply:
When doing the mounts in the guest (instead of on the host), it needs to know if they are read-write or read-only

Reply:
An alternative would be to do the mounts over ssh with `sudo mount`, similar to how we run the `sshfs` mount now